pyquil/api/_compiler.py

```diff
@@ -50,6 +51,19 @@ class QPUCompilerNotRunning(Exception):
     pass
 
 
+def refresh_client(client: Client, new_endpoint: str) -> Client:
```

Comment:
Here.

Reply:
That's the QPU compiler's client, not the QPU's client

Reply:
Why have two (or five or six) functions that do exactly the same thing, when you could just have one?